pkg/config/config.go

```diff
@@ -339,20 +335,6 @@ type Bundle struct {
 	Targets    []string `yaml:"targets,omitempty"`
 }
 
-const DefaultHeader = "#cloud-config"
-
-func HasHeader(userdata, head string) (bool, string) {
```

Comment:
https://github.com/kairos-io/provider-kairos/blob/5b9bdda87d290893ef58626a12585f373608e985/internal/cli/token/rotate.go#L69

Reply:
mmmh should be implemented there on in the sdk, makes no sense to have a function here that its used bu another library lool